kayenta-standalone-canary-analysis/src/main/java/com/netflix/kayenta/canaryanalysis/domain/CanaryAnalysisExecutionRequest.java

```diff
@@ -80,6 +81,12 @@
       "If this is set to a value greater than the lifetime, it will be reset to lifetime.")
   private Long analysisIntervalMins;
 
+  @ApiModelProperty(value =
+      "A map of customizable data that among other things can be used in org specific external modules such as event " +
+      "listeners to handle notifications such as Slack, email, async http callbacks, etc.\n" +
+      "The contents of this field doesn't have an effect on the actual canary analysis execution")
```

Comment:
s/doesn't/don't

And missing trailing period.

Reply:
- [X] done